riff-raff.yaml

```diff
@@ -14,7 +14,7 @@ deployments:
   delete-expired-formstack-data-lambdas:
     type: aws-lambda
     parameters:
-      bucket: delete-expired-formstack-data-dist
+      bucketSsmLookup: true
```

Comment:
This app is deployed into the `flexible` stack, which [sits in the Composer account](https://riffraff.gutools.co.uk/deployinfo/data?key=credentials%3Aaws-cfn-role#:~:text=cfn%20execute%20role-,flexible,-.*).

The default SSM Parameter is `/account/services/artifact.bucket`, which doesn't match "delete-expired-formstack-data-dist":

```bash
aws ssm get-parameter \
  --name /account/services/artifact.bucket \
  --profile composer \
  --region eu-west-1 \
  | jq -r '.Parameter.Value'
```

I'd recommend either:
  - Update [the CFN](https://github.com/guardian/delete-expired-formstack-data/blob/main/template.yaml#L36) to use the bucket name on `/account/services/artifact.bucket`
  - OR set the `bucketSsmKey` parameter to use a different SSM key